redisdb/tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import subprocess
+import os
+import time
+
+import pytest
+import redis
+
+from .common import HOST, PORT, MASTER_PORT, REPLICA_PORT, PASSWORD
+
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def wait_for_cluster(master, replica):
```

Comment:
We should use docker `healthcheck` instead. This would make our tests faster, self contained and more reliable

Reply:
They removed `depends_on: condition` from compose, in this situation I could add an health check to the master checking if a slave connected, but this would mean moving this python snippet to a shell script, is it worth?

Reply:
Didn't knew they removed `condition` from docker-compose 3 ...

We could keep the python code or use version 2.1 of compose and use something like this for the healthcheck on masrter:
```bash
redis-cli ping
```

I don't know if it's worth it since it's deprecated. But having a standalone compose file is very helpful when debugging/developing.

Reply:
This reminds me I'd also need to install `redis-cli` on the host :/
BTW, `ping` wouldn't be enough because the `master` could pass the health check before a slave was connected (and the latter is a requirement for some tests).